build/Targets/VSL.Versions.targets

```diff
@@ -73,8 +73,8 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <CommonNuGetMoniker>beta2</CommonNuGetMoniker>
-    <VSSemanticVersion>15.3</VSSemanticVersion>    
+    <CommonNuGetMoniker>beta3</CommonNuGetMoniker>
```

Comment:
The `1.0.0-*` at Line 80 and 82 will trigger in the case of local Builds only i.e. if the `'$(BuildNumberFiveDigitDateStamp)' != '' AND '$(BuildNumberBuildOfTheDayPadded)' != ''` condition is false. We should document it.

To get the `15.6` prefix, the value of the `RoslynSemanticVersion` variable(at the top of the file) should be changed. This will give us the version `15.6-beta1`